projects/unit/project.yaml

```diff
@@ -0,0 +1,16 @@
+homepage: "https://unit.nginx.org"
+language: c
+primary_contact: "security-alert@nginx.org"
+vendor_ccs:
+  - "unit@nginx.org"
+auto_ccs:
```

Comment:
Please add "devrep@nginx.com" (same as for the `njs` project for convenience) to the `auto_ccs`.

Reply:
Note that only people with accounts in auto_cc or primary_contact will be able to access the testcases